ha/test/integration/health_monitor/test_event_action_handler.py

```diff
@@ -43,7 +44,7 @@ def receive(message):
         event_manager = EventManager.get_instance()
         component = SUBSCRIPTION_LIST.CSM
         resource_type = RESOURCE_TYPES.NODE
-        state = RESOURCE_TYPES.OFFLINE
+        state = HEALTH_STATUSES.OFFLINE
```

Comment:
I don't think so, OFFLINE is a supported health status now for any of the resource. Please use "failed" or any one applicable.